bcftools.wdl

```diff
@@ -0,0 +1,51 @@
+version 1.0
+
+# MIT License
+#
+# Copyright (c) 2018 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task Bcf2Vcf {
+    input {
+        File bcf
+        String outputPath
+        String dockerImage = "quay.io/biocontainers/bcftools:1.9--ha228f0b_3"
+    }
+
+    command {
+        set -e
+        mkdir -p "$(dirname ~{outputPath})"
+        bcftools view ~{bcf} -O v -o ~{outputPath}
+    }
+
+    output {
+        File outputVcf = outputPath
+    }
+
+    runtime {
+        docker: dockerImage
+    }
+
+    parameter_meta {
+        bcf: {description: "The generated BCF from an SV caller", category: "advanced"}
+        outputPath: {description: "The location the output VCF file should be written.", category: "common"}
```

Comment:
These are required inputs.

Reply:
The BCF was generated by Delly, isn't that advanced?